js/apps/admin-ui/src/clients/credentials/Credentials.tsx

```diff
@@ -200,10 +204,8 @@ export const Credentials = ({ client, save, refresh }: CredentialsProps) => {
               </Button>
             </ActionGroup>
           </CardBody>
-          {(clientAuthenticatorType === "client-secret" ||
-            clientAuthenticatorType === "client-secret-jwt") && <Divider />}
-          {(clientAuthenticatorType === "client-secret" ||
-            clientAuthenticatorType === "client-secret-jwt") && (
+          {selectedProvider?.supportsSecret && <Divider />}
+          {selectedProvider?.supportsSecret && (
```

Comment:
Looks like we can probably have just one statement here and render both the `Divider` and the `CardBody`, not sure why it wasn't like that in the first place.

Reply:
Okay. I will fix that.